.github/workflows/build.yml

```diff
@@ -1,24 +1,30 @@
 name: Build
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+
     steps:
-    - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ env.NODE_VERSION }}
 
-    - name: Use Node.js ${{ env.NODE_VERSION }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ env.NODE_VERSION }}
+      - name: npm install, and test (ubuntu)
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        run: |
+          npm install
```

Comment:
extremely minor nit but we could dry this up by making `npm install` its own step before branching out into the different OS steps for the test command

Reply:
Good suggestion, I'll do that.